data/card_catalogue.json

```diff
@@ -503,19 +555,19 @@
         {
           "title": "Bound Bow",
           "upgrade": "",
-          "tier": "blue",
+          "tier": "green",
           "details": {
-            "rules": "You summon a bow made of light. Deal triple damage on your next ranged attack.",
+            "rules": "While hodling this card in your hand, roll an additional d4 when you perform a ranged attack. Cast to deal 6 damage to a visible target.",
             "flavour": "They stood on the hill for a moment, mesmerized by the dazzling light. -Battlefield Diaries, General Polonius",
             "image": "assets/core/card/illustration/Bound Bow.png"
           }
         },
         {
           "title": "Bound Sword",
           "upgrade": "",
-          "tier": "blue",
+          "tier": "green",
           "details": {
-            "rules": "You summon a sword made of light. Deal triple damage on your next melee attack.",
+            "rules": "While holding this card in your hand, roll an additional d6 when you perform a melee attack. Cast to deal 6 damage to a visible target.",
```

Comment:
Changing these to follow the weapon pattern.

![image](https://user-images.githubusercontent.com/8670351/116803872-fdac6380-aae8-11eb-98a4-9a6b8a210e6c.png)

It's kinda wordy as is, I think for these we should just cut out the flavour text 🤔

Reply:
Changes look good, shorter flavour text >> none, but here the amount of text on the card is no problem at all.

Reply:
Yeah I'll rejig the flavour text. Probably just cutting out the attribution of the quote will do it here.

Reply:
Thrown up a ticket for that, plus we've got the keyword/emoji thread to talk further about simplifying this text through keywords